magenta/models/image_stylization/README.md

```diff
@@ -53,33 +53,30 @@ $ image_stylization_train \
       --style_dataset_file=/tmp/image_stylization/style_images.tfrecord \
       --num_styles=<NUMBER_OF_STYLES> \
       --vgg_checkpoint=/path/to/vgg_16.ckpt \
-      --imagenet_data_dir=/path/to/imagenet-2012-tfrecord \
-      --style_dataset_file=/tmp/image_stylization/style_images.tfrecord
+      --imagenet_data_dir=/path/to/imagenet-2012-tfrecord
 ```
 
 To evaluate the model:
 
 ```bash
 $ image_stylization_evaluate \
       --style_dataset_file=/tmp/image_stylization/style_images.tfrecord \
-      --train_dir=/tmp/image_stylization/run1/train \
-      --eval_dir=/tmp/image_stylization/run1/eval \
+      --train_dir=/tmp/image_stylization/train/run1 \
```

Comment:
Our melody_rnn model follows the pattern of run1/train. Is there a reason to do this differently?